homeassistant/components/esphome/bluetooth/client.py

```diff
@@ -56,38 +57,40 @@ def mac_to_int(address: str) -> int:
     return int(address.replace(":", ""), 16)
 
 
+def _on_disconnected(task: asyncio.Task[Any], fut: asyncio.Future[None]) -> None:
```

Comment:
Personally, I don't like abbreviations in code, unless they are _super_ obvious. So to me, looking at this, is this a futex? or future?. Would it hurt naming the (unused dummy) variable something slightly more descriptive?

Reply:
Unused argument.

```suggestion
def _on_disconnected(task: asyncio.Task[Any], _: asyncio.Future[None]) -> None:
```